cli/src/main/resources/help/schema.adoc

```diff
@@ -5,23 +5,50 @@ SCHEMA(1)
 
 NAME
 ----
-schema - uploads and registers protobuf schemas.
+schema - manipulates protobuf schemas.
 
 
 SYNOPSIS
 --------
-*schema* ['OPTIONS'] `SCHEMA_NAME`
 
+*schema ls*
 
-OPTIONS
--------
-*-u, --upload*='FILE'::
+*schema upload --file=/path/to/schema.proto schema.proto*
+
+*schema remove schema.proto*
+
+*schema get schema.proto*
+
+DESCRIPTION
```

Comment:
needs 2 new lines otherwise, the format is broken. `help schemas` show:
```
  schema get schema.proto
  DESCRIPTION
    ----------- 
 
  Manage schemas with the `ls`, `upload`, `get`, `remove` subcommands.
```

Reply:
Fixed